.github/workflows/deploy-cloudflare.yml

```
@@ -56,3 +56,329 @@ jobs:
+          persist-credentials: false
+
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+          cache: pip
+
+      - name: Install httpx + requests
+        run: pip install httpx requests
+
+      - name: Smoke: public health endpoints
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)